int8 new_flowrate
---
bool success